packages/web/hooks/one-click-trading/use-one-click-trading-session.ts

```diff
@@ -67,12 +86,15 @@ export const useOneClickTradingSession = ({
     const timeoutId = setTimeout(() => {
       if (!value?.info) return;
 
-      setIsExpired(true);
+      // trigger mobx store to update when session expires
+      // (isExpired is a computed value in the store)
+      runInAction(() => {});
+
```

Comment:
it would introduce more problems than it would solve

Storing the expired flag in the store would trigger reactivity but would be used only as a trigger since the isExpired needs to be calculated. This would introduce a prop that's persisted and tracked without any real usage.